lib/tasks/demo.js

```diff
@@ -258,8 +258,10 @@ module.exports = function(gulp, config) {
 		}
 
 		demosConfig.demos.forEach(function(demoConfig) {
-			// Extend an empty object to avoid changing the value of defaultDemoConfig
-			demos.push(extend(true, {}, defaultDemoConfig, demosConfig.demosDefaults || demosConfig.options, demoConfig));
+			if (!config.buildDemo || (config.buildDemo && config.buildDemo === demoConfig.name)) {
```

Comment:
Not sure on the naming (sorry). `buildDemo` feels like an action or a task to me, not a config option.

All OBT options are global, right, not scoped to their task?  Could we call it `demoFilter`, maybe?

Also needs adding to README.